exampleplugins/5.2/SwagProductAssoc/Resources/menu.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/menu.xsd">
+
+    <entries>
+        <entry>
+            <name>SwagProductAssoc</name>
+            <label lang="en">Swag product association</label>
+            <label lang="de">Swag Produkt Assoziation</label>
+            <controller>SwagProductAssoc</controller>
+            <action>index</action>
+            <class>sprite-application-block</class>
+            <parent identifiedBy="label">Artikel</parent>
```

Comment:
please do not identify by label, because on english installations, it will not work :wink: